great-idea/index.html

```diff
@@ -14,10 +15,112 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <div class="index-container">
+    <header>
+      <div class="nav-content-container">
+        <nav class="nav-bar">
+          <section class="nav-links">
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </section>
```

Comment:
in this header area, you don't need to use section tags, you want to use section tags more specifically for your content